publisher/publisher_test.go

```diff
@@ -198,23 +118,43 @@ func errorBodyLogSrv() *httptest.Server {
 	return server
 }
 
-func setup(t *testing.T) (*Impl, *x509.Certificate, *ecdsa.PrivateKey) {
-	intermediatePEM, _ := pem.Decode([]byte(testIntermediate))
-
-	pub := New(nil,
+func setup(t *testing.T) (*Impl, []*x509.Certificate, *ecdsa.PrivateKey) {
+	// Load a chain with two intermediates
+	testCA2Cert, err := core.LoadCert("../test/test-ca2.pem")
+	test.AssertNotError(t, err, "Unable to load ../test/test-ca2.pem")
+
+	testCA2CrossCert, err := core.LoadCert("../test/test-ca2-cross.pem")
+	test.AssertNotError(t, err, "Unable to load ../test/test-ca2-cross.pem")
+
+	// Create our ct.ASN1Cert bundles mapped to their corresponding
+	// IssueNameID
+	issuerBundles := map[issuance.IssuerNameID][]ct.ASN1Cert{
+		// Chain with two intermediates
+		issuance.IssuerNameID(18337263084599622): {
+			ct.ASN1Cert{Data: testCA2Cert.Raw},
+			ct.ASN1Cert{Data: testCA2CrossCert.Raw},
+		},
+		// Chain with two intermediates mapped to the IssuerNameID of
+		// publisher/test/178.pem
+		issuance.IssuerNameID(66191037641995744): {
+			ct.ASN1Cert{Data: testCA2Cert.Raw},
+			ct.ASN1Cert{Data: testCA2CrossCert.Raw},
+		},
+	}
```

Comment:
Since you have tests for `issuance.LoadChain` and `getBundleForChain` elsewhere, you could just take advantage of those here to streamline all this manual bundle construction.